src/display/api.js

```diff
@@ -407,6 +420,7 @@ function _fetchDocument(worker, source, pdfDataRangeTransport, docId) {
     });
 }
 
+/** @lends <global> */
```

Comment:
We're already exposing a bunch interfaces for the unit-tests, hence please try to see if simply listing `PDFDocumentLoadingTask` with the exports at the bottom of the file works instead.

Reply:
Exporting a `const` that happens to be assigned to a `class` does not mean that class get's exported. And I don't know if there's any way to export the `class` itself from within the closure.